static-assets/scripts/admin.js

```diff
@@ -549,16 +549,22 @@
                 }, audit.defaultDelay);
             };
 
-            audit.updateSite = function(site){
+            audit.updateSite = function(site, isNoTranslate){
```

Comment:
Rename `isNoTranslate` to better indicate what it is. The logic now does the opposite of what I thought it’d do based on the name of the var.

Reply:
Done